cli/generate/src/dsl.js

```diff
@@ -466,6 +466,7 @@ function checkPrecedence(value) {
 }
 
 function getEnv(name) {
+  if (globalThis.native) return __ts_grammar_path;
```

Comment:
We should add an env module and use it here (or just use quickjs's `std.getenv` if rquickjs allows that).